capa/features/extractors/vmray/call.py

```diff
@@ -30,34 +32,34 @@ def extract_call_features(ph: ProcessHandle, th: ThreadHandle, ch: CallHandle) -
 
     # TODO (meh): update for new models https://github.com/mandiant/capa/issues/2148
     # print(ch)
-    return
+    
 
     # Extract API name
-    yield API(ch.inner.name), ch.inner.address
+    yield API(fncall.name), Address(fncall.address)
 
     # Extract arguments from <in>
-    for param in ch.inner.in_:
-        value = param.value
-        if isinstance(value, str):
-            yield String(value), ch.inner.address
-
-        elif isinstance(value, int):
-            yield Number(value), ch.inner.address
-
-        else:
-            assert_never(value)
+    if fncall.in_ is not None:
+        for param in fncall.in_.params:
```

Comment:
I don't see these fields (`in_`, `in_.params`) described in the [`FunctionCall`](https://github.com/r-sm2024/vmray/blob/0676b2480da7f6a09047e00d64d093805de2cb3a/capa/features/extractors/vmray/models.py#L15-L22) model. Does the model need to be updated? Same feedback for the fields used later (`out_`, `out_.params`).

Reply:
Please add the fields to cover this, example data:
```yml
        "in": {
          "param": {
            "name": "ProcessorFeature",
            "type": "unsigned_32bit",
            "value": "0xa"
          }
        },
        "out": {
          "param": {
            "name": "ret_val",
            "type": "bool",
            "value": "1"
          }
        }
```

Reply:
the types should be Literals from this list, but we probably have to run some tests to confirm these and/or to find additional ones

```
unknown
void
bool
signed_8bit
unsigned_8bit
signed_16bit
unsigned_16bit
signed_32bit
unsigned_32bit
signed_64bit
unsigned_64bit
double
void_ptr
ptr
str
array
container (structs etc.)
bindata (binary data)
undefined_type
```

Reply:
> I don't see these fields (`in_`, `in_.params`) described in the [`FunctionCall`](https://github.com/r-sm2024/vmray/blob/0676b2480da7f6a09047e00d64d093805de2cb3a/capa/features/extractors/vmray/models.py#L15-L22) model. Does the model need to be updated? Same feedback for the fields used later (`out_`, `out_.params`).

I think this is leftover from an edit I made awhile back. I've added the in_ and out params to models.py.

Question - Can pydantic detect that in_ and out are dictionaries and automatically assume and match key names to values? For example, I added this to the FunctionCall class in models.py:

in_: Dict = {}
out_: Dict = {}

Would I have to explicitly specify aliases for the "name", "type" and "value" from  

"in": {
          "param": {
            "name": "ProcessorFeature",
            "type": "unsigned_32bit",
            "value": "0xa"
          }
        },
        "out": {
          "param": {
            "name": "ret_val",
            "type": "bool",
            "value": "1"
          }
        }